vendor/blockchain_contracts/src/bin/calculate_offsets/ethereum/rfc003/contract.rs

```diff
@@ -0,0 +1,79 @@
+extern crate regex;
+use crate::calculate_offsets::ethereum::rfc003::{
+    compile_contract::compile, metadata::Metadata, offset::Offset,
+    placeholder_config::PlaceholderConfig, Error,
+};
+
+const TEMPLATE_FOLDER: &str = "./src/bin/calculate_offsets/ethereum/rfc003/templates/";
+const CONTRACT_FILE: &str = "contract.asm";
+const HEADER_FILE: &str = "deploy_header.asm";
+const CONFIG_FILE: &str = "config.json";
+
+pub struct Contract {
+    bytes: Vec<u8>,
+    placeholder_config: PlaceholderConfig,
+    pub meta_data: Metadata,
+}
+
+impl Contract {
+    pub fn compile_from_directory_and_load_placeholder_config(
```

Comment:
:wrench: 
I would suggest renaming to just `compile` because the fact that you are using a config file is kind of an implementation detail, as the caller, I don't really care about that + I also don't get much choice about whether it happens or not.

Reply:
done